datasafe-rest-impl/datasafe.postman_collection.json

```diff
@@ -0,0 +1,690 @@
+          },
+          {
+            "key": "password",
+            "value": "ver",
+            "type": "text"
+          },
+          {
+            "key": "token",
+            "value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJzZWN1cmUtYXBpIiwiYXVkIjoic2VjdXJlLWFwcCIsInN1YiI6InVzZXJuYW1lIiwiZXhwIjoxNTYxMzA0MTA4LCJyb2wiOlsiUk9MRV9VU0VSIl19.Ungu5hob8mbwb7GfWfvqj1VsrNNv1pN6UEMJYJHSAHBicRlsYVbRAE7Ju_N1nnjdoZ6TSs4hireff92SkXvesQ",
+            "type": "text"
+          },
+          {
+            "key": "Accept",
+            "value": "application/json",
+            "type": "text"
+          },
+          {
+            "key": "Content-Type",
+            "value": "application/json",
+            "type": "text"
+          }
+        ],
+        "url": {
+          "raw": "{{host}}/versioned/test4.txt",
+          "host": [
+            "{{host}}"
+          ],
+          "path": [
+            "versioned",
+            "test4.txt"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "versioned write",
+      "request": {
+        "method": "PUT",
+        "header": [
+          {
+            "key": "Content-Type",
+            "value": "application/octet-stream",
+            "type": "text"
+          },
+          {
+            "key": "user",
+            "value": "ver",
+            "type": "text"
+          },
+          {
+            "key": "password",
+            "value": "ver",
+            "type": "text"
+          },
+          {
+            "key": "token",
+            "value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJzZWN1cmUtYXBpIiwiYXVkIjoic2VjdXJlLWFwcCIsInN1YiI6InVzZXJuYW1lIiwiZXhwIjoxNTYxMzA0MTA4LCJyb2wiOlsiUk9MRV9VU0VSIl19.Ungu5hob8mbwb7GfWfvqj1VsrNNv1pN6UEMJYJHSAHBicRlsYVbRAE7Ju_N1nnjdoZ6TSs4hireff92SkXvesQ",
+            "type": "text"
+          }
+        ],
+        "body": {
+          "mode": "file",
+          "file": {
+            "src": "/Users/maxim/parallel_transactions_merge_logic_fix.patch"
+          }
+        },
+        "url": {
+          "raw": "{{host}}/versioned/test10.txt",
+          "host": [
+            "{{host}}"
+          ],
+          "path": [
+            "versioned",
+            "test10.txt"
+          ]
+        }
+      },
+      "response": []
+    },
+    {
+      "name": "versioned list",
+      "request": {
+        "method": "GET",
+        "header": [
+          {
+            "key": "user",
+            "value": "ver",
+            "type": "text"
+          },
+          {
+            "key": "password",
+            "value": "ver",
+            "type": "text"
+          },
+          {
+            "key": "token",
+            "value": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJzZWN1cmUtYXBpIiwiYXVkIjoic2VjdXJlLWFwcCIsInN1YiI6InVzZXJuYW1lIiwiZXhwIjoxNTYxMzA0MTA4LCJyb2wiOlsiUk9MRV9VU0VSIl19.Ungu5hob8mbwb7GfWfvqj1VsrNNv1pN6UEMJYJHSAHBicRlsYVbRAE7Ju_N1nnjdoZ6TSs4hireff92SkXvesQ",
```

Comment:
I hope with this token you can't get  to REST app

Reply:
you can, but token expired